core/src/main/java/hudson/model/AbstractItem.java

```diff
@@ -434,7 +434,7 @@ protected void renameTo(final String newName) throws IOException {
                             Util.deleteRecursive(oldRoot);
                         } catch (IOException e) {
                             // but ignore the error, since we expect that
-                            e.printStackTrace();
+                            LOGGER.log(Level.FINEST, "Ignoring IOException while deleting", e);
```

Comment:
```suggestion
                            LOGGER.log(Level.WARNING, "Ignoring IOException while deleting", e);
```
though this code seems pretty fragile to begin with.